corehq/apps/hqwebapp/static/hqwebapp/js/bootstrap3/ui-element.js

```diff
@@ -1,58 +0,0 @@
-/* globals hqImport, hqDefine */
-
-hqDefine('hqwebapp/js/bootstrap3/ui-element', function () {
-    'use strict';
-    var module = {};
-
-    module.input = function (value) {
-        return hqImport('hqwebapp/js/ui_elements/bootstrap3/ui-element-input').new(value);
-    };
-
-    module.textarea = function () {
-        return hqImport('hqwebapp/js/ui_elements/bootstrap3/ui-element-input').new_textarea();
-    };
-
-    module.select = function (options) {
-        return hqImport('hqwebapp/js/ui_elements/bootstrap3/ui-element-select').new(options);
-    };
-
-    module.map_list = function (guid, modalTitle) {
-        return hqImport('hqwebapp/js/ui_elements/bootstrap3/ui-element-key-val-list').new(guid, modalTitle);
-    };
-
-    module.input_map = function (show_del_button) {
-        return hqImport('hqwebapp/js/ui_elements/bootstrap3/ui-element-input-map').new(show_del_button);
-    };
-
-    module.checkbox = function () {
-        return hqImport('hqwebapp/js/ui_elements/bootstrap3/ui-element-checkbox').new();
-    };
-
-    module.langcode_tag_btn = function ($elem, new_lang) {
-        return hqImport('hqwebapp/js/ui_elements/ui-element-langcode-button').new($elem, new_lang);
-    };
-
-    module.key_value_mapping = function (options) {
-        return hqImport('hqwebapp/js/ui_elements/bootstrap3/ui-element-key-val-mapping').new(options);
-    };
-
-    module.serialize = function (obj) {
```

Comment:
Was this unused? I didn't see anywhere where it is replaced in this PR.

Reply:
Yes, this was unused, though I didn't dig around to see when its last usage was removed.